src/controllers/user.controller.js

```
@@ -1,106 +1,552 @@
+ *       - bearerAuth: []
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         schema:
+ *           type: string
+ *         required: true
+ *         description: ID của người dùng cần cập nhật
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)